lib/rust/frp/src/stream.rs

```diff
@@ -14,45 +14,53 @@ use crate::data::watch;
 
 /// Owned call stack type.
 #[cfg(feature = "stack-trace")]
-pub type OwnedCallStack = EnabledCallStack;
+pub type OwnedCallStack<'a> = EnabledCallStack<'a>;
```

Comment:
Perhaps we should rename this? If it keeps a lifetime, it isn't entirely "Owned", is it?

Reply:
You can own a struct with a reference :) But I agree, it is a bit inaccurate. I'm going to get rid of this type entirely, as there is no point to refer to that callstack struct as "owned" anyway. If we ever need that, we will need to flatten it into a vec.